### ISSUE TITLE
jax-releases dependencies only for cuda* for linux

### ISSUE THREAD

**Issue by GraoMelo** [xai-org/grok-1#328]
Hello, congratulations on the initiative to share the project on GitHub.
I would like to suggest that the hardware requirements be explicitly stated in the documentation.

When analyzing the dependencies:

jax[cuda12-pip]==0.4.25 -f https://storage.googleapis.com/jax-releases/jax_cuda_releases.html

In my view, it makes sense, but on the URL:

https://storage.googleapis.com/jax-releases/jax_cuda_releases.html

there are many files:

cuda100/jaxlib-0.1.52+cuda100-cp36-none-manylinux2010_x86_64.whl
cuda101/jaxlib-0.1.52+cuda101-cp36-none-manylinux2010_x86_64.whl
cuda102/jaxlib-0.1.52+cuda102-cp36-none-manylinux2010_x86_64.whl
cuda11/jaxlib-0.1.73+cuda11.cudnn805-cp37-none-manylinux2010_x86_64.whl

or even:

cuda110/jaxlib-0.1.52+cuda110-cp36-none-manylinux2010_x86_64.whl
cuda111/jaxlib-0.1.56+cuda111-cp36-none-manylinux2010_x86_64.whl

or still

cuda112/jaxlib-0.1.65+cuda112-cp39-none-manylinux2010_x86_64.whl

cuda12/jaxlib-0.4.10+cuda12.cudnn88-cp310-cp310-manylinux2014_x86_64.whl

All the files have the suffix:

manylinux20yy = yy year year
_x86_64.whl = X86 or amd64 architecture

In other words, little compatibility with Unix or Windows.
I'm not saying it's not possible to emulate via WSL or even containers or virtual machines, but it's not written anywhere that the support is only for Linux when using the CUDA modules.
